docs/product-installation-guide/src/main/asciidoc/controller-standalone-con.adoc

```diff
@@ -0,0 +1,6 @@
+[id='controller-standalone-con']
+= Installing the {KIE_SERVER} standalone controller
+
+You can configure {KIE_SERVER} to run in managed or unmanaged mode. If {KIE_SERVER} is unmanaged, you must manually create and maintain containers. If {KIE_SERVER} is managed, the {KIE_SERVER} controller manages the {KIE_SERVER} configuration and you interact with the controller to create and maintain containers. 
+
+The controller is integrated with {CENTRAL}. If you install {CENTRAL}, use the *Exection Server* page to create and maintain containers. However, if you do not install {CENTRAL}, you can install the standalone {KIE_SERVER} controller and use the REST API or the {KIE_SERVER} Java Client API to interact with it.
```

Comment:
"{KIE_SERVER} Java Client API" should be something like "standalone {KIE_SERVER} controller Java Client API".

Btw is "standalone {KIE_SERVER} controller" the official name for Standalone Controller? I hear it for the first time.

Reply:
After speaking with Ivo again, we decided that this is acceptable:
standalone Decision Server Controller.